src/test/java/john/commands/FindCommandTest.java

```diff
@@ -0,0 +1,29 @@
+package john.commands;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
+
+import john.data.TaskList;
+import john.ui.Ui;
+
+public class FindCommandTest {
+    @Test
+    public void executeTestNoResults() {
```

Comment:
Perhaps you can consider setting the test methods in the following format?
https://se-education.org/guides/conventions/java/intermediate.html

![image](https://user-images.githubusercontent.com/50956270/188148781-7ee9f1e3-5a34-4eb7-896d-0619e57c2ef0.png)

Reply:
I'm working on this, thanks for the suggestion!